### DISCUSSION TITLE
How to get the search in the sidebar?

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#832] [Q&A] [answered]
How do I enable the Search panel in the sidbar or is it possible to get the search bar back that was previously in the top bar?

Searched in https://hbs.razonyang.com/v1/en/docs/configuration/site-params/ and also https://github.com/razonyang/hugo-theme-bootstrap/blob/master/exampleSite/config/_default/params.toml but couldn't find anything.

**Comment by razonyang** [comment]
It was removed, I'll bring it back later.

**Comment by razonyang** [comment, marked as answer]
Reverted via #833, if site build fails, please upgrade the dependencies via `hugo mod npm pack` and then `npm update`.

> Since the upstream branch `github:twbs/bootstrap#new-masthead-darkmode` was removed, we'll need to change it as `github:twbs/bootstrap#main` (`package.json`). `hugo mod npm pack` should take care of this, you can also change this manually if it doesn't.

BTW, you include the search form on the sidebar via hook `sidebar-begin`:

https://github.com/razonyang/hugo-theme-bootstrap/blob/ad3b22d99f95caa82a6ae0b27d162f0b3f0794f5/exampleSite/layouts/partials/hooks/sidebar-begin.html#L1
